components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1074,18 +1078,22 @@ interface KeyringService {
   ImportAccount(string accountName, string privateKey, CoinType coin)
     => (AccountInfo? account);
 
-  // Imports a filecoin account with a specific private key and network
+  // Imports a filecoin account with a specific private key and network.
   ImportFilecoinAccount(string accountName, string privateKey, string network)
       => (AccountInfo? account);
 
-  // Imports an account via JSON
+  // Imports a bitcoin account for a network. Payload is an extended private key.
+  ImportBitcoinAccount(string accountName, string payload, string network)
+      => (AccountInfo? account);
```

Comment:
Should `string payload` be `string privateKey` to align with other import functions? Or is it payload to support other types in future?

Reply:
That is not exactly a private key now, but an extended private key. Also most likely we'll want to support `xpub` extended public keys for watch-only accounts.